Shared/LocationDisplay3d.cpp

```diff
@@ -164,7 +161,7 @@ void LocationDisplay3d::setPositionSource(QGeoPositionInfoSource* positionSource
 
     m_headingConnection = connect(gpxLocationSimulator, &GPXLocationSimulator::headingChanged, this, [this](double heading)
     {
-      m_locationGraphic->attributes()->replaceAttribute(s_headingAttribute, heading);
+      m_defaultSymbol->setAngle(static_cast<float>(-heading));
```

Comment:
This is fine - but we should confirm if there is a way to do this with the attributes of the element or similar